### PR TITLE
doc: a better README

### PR DESCRIPTION
**Pull request by mintbomb27**
Hey! I've tried making a few changes to the readme for it to look good, and corrected some mistakes. Please look into it. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
-A cross platform tool which instantly notifies about COVID vaccine availability.
+
+# 💉 CoWIN Notifier 😷
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/aryan9600/cowin-notifier)
+[![GitHub License](https://img.shields.io/github/license/aryan9600/cowin-notifier)](https://github.com/aryan9600/cowin-notifier/blob/master/LICENSE)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/aryan9600/cowin-notifier/issues/new/choose)
+![release](https://img.shields.io/github/v/release/aryan9600/cowin-notifier)
+![rust-version](https://img.shields.io/badge/rust-v1.5.2-red)
+
+A cross-platform tool written in rust, which instantly notifies users about COVID-19 vaccine availability at their regions.
```
Comment (DarthBenro008) [inline README.md]: I think it would be nice if we could mention this tool is compatible only in India

File: README.md [hunk 2]
```diff
@@ -1,40 +1,46 @@
...
-* Run `cd ~/Downloads`
-
-* Run `./cowin-notifier "state_name", "district_name", "age_group"`
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2016.06.28.png)
-
+* Open your terminal and `cd` into your default Downloads directory where you have downloaded the file. macOS users can fire up Spotlight using `cmd + Space`, search for Terminal and open it.
+![macOS Screenshot-1](https://github.com/mintbomb27/cowin-notifier/raw/main/assets/macOS%20spotlight.png)
+* Run `cd ~/Downloads` 
+* Run `chmod +x cowin-notifier`
+* Run `./cowin-notifier "state_name" "district_name" "age_group"`
```
Comment (DarthBenro008) [inline README.md]: I think it would look a bit better if they are block commands and eliminate the redundant `Run` word in each step, something similar to:

* Run the following commands:
```
cd ~/Downloads
chmod +x cowin-notifier
./cowin-notifier "state_name" "district_name" "age_group"
```

File: README.md [hunk 3]
```diff
@@ -1,40 +1,46 @@
...
+![macOS Screenshot-2](https://github.com/mintbomb27/cowin-notifier/raw/main/assets/macOS%20notif.png)
 * Minimize your terminal.
 * If a slot opens up for your age group and district, you'll recieve a notification. __Make sure that you don't close the Terminal.__
 
-![Alt text](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2017.46.10.png)
+![macOS Screenshot-3](https://raw.githubusercontent.com/aryan9600/cowin-notifier/main/assets/Screen%20Shot%202021-05-05%20at%2017.46.10.png)
 
-### Linux
+### 🐧 Linux
 * We use the [reqwest](https://github.com/seanmonstar/reqwest) crate, which has some specific [requirements for Linux](https://github.com/seanmonstar/reqwest#requirements). Please follow them first.
```
Comment (DarthBenro008) [inline README.md]: Could you also please rephrase this asking the user to install the requirements and dependencies instead of  `Please follow them first`

**Review by DarthBenro008 — CHANGES_REQUESTED** [review]
Please check out the comments and also kindly remove `.DS_Store` files from the folders.

Rest lgtm. Thank you for opening a PR 😃 !

File: README.md (hunk 2 above)
Comment (aryan9600) [reply README.md]: Also, don't think you need to `chmod` the binary.

Comment (mintbomb27) [reply README.md]: I actually tried at first without the chmod, but didn't work. That is why I added it.

Comment (DarthBenro008) [reply README.md]: @aryan9600 I believe some unzippers preserve the state permission and have safe default permission set to 666 or 644. Hence it's a safe option to` chmod +x` it. However, if #4 gets merged. This step wouldn't be required anymore and we can also shift to a bash script that places the binary in user's `bin` folder.

**Comment by mintbomb27** [comment]
I have committed after applying the requested changes

**Comment by DarthBenro008** [comment]
lgtm! Thanks for contributing! 😃
